README.md

```diff
@@ -68,6 +68,21 @@ class UserFactory extends BaseFactory
 
 Inside this class, you can define the properties of the model with the `getData` method. It is very similar to what you would do with a Laravel default factory, and you can make use of Faker as well. The `create` method is only a copy of the one in the parent class `BaseFactory`. Still, we need it in our dedicated factory class so that we can define what gets returned. In our case, it is a user model. Other methods like `new` or `times` are hidden in the parent class.
 
+### Additional arguments and options
+
+If you don't want to select your model from a list, you can pass the class name of a model in your model path as an argument and your factory will immediately be created for you:
+```php
+php artisan make:factory-reloaded Ingredient
+```
+By default, this command will stop and give you an error if a factory you're trying to create already exists. You can overwrite an existing factory using the force option:
+```php
+php artisan make:factory-reloaded Ingredient --force
+```
+You can override the configurations as options as well:
+`--models_path=path/to/your/models`
```

Comment:
I noticed that it is possible to pass a model `and` an option like the `models_path`. I'm not sure if we can prevent this but my first thought was that, it will look for the model in the given path:

e.g. `php artisan make:factory-reloaded Ingredient --models_path=app/Models

But it just ignores the option in this case right?

Reply:
I honestly hadn't thought of that. I didn't test for it so I don't know how it actually works as written. I'll add a test to look for the model a level deeper than where they currently sit and I'll use the path to make that work. The correct behavior would be for it to look for Ingredient inside of app/Models. Once I've done that I'll add an example reflecting it to the README as requested above. I'll dig back into it this evening. Day job is calling for now. :)

Reply:
sounds great. Thanks a lot 👍

Reply:
The only challenge I'm having now is getting the fully qualified class name of the model. I'd like to use getFullyQualifiedClassNameFromFile() from the ClassFinder class in your really cool class picker package like you do when a user selects a model. I can't find a public method that I can simply pass a filename to that will reach into that method. I could either create a new class that extends ClassFinder or we can add a public method into that class that will let us just pass an arbitrary filename in and get the fqclassname back. How would you prefer to approach it?

Honestly, my preference would be to add a public method to the ClassFinder class that could reach into the protected method because I can think of about 10 other uses outside this package for that functionality. I'd love to borrow it for at least 2 packages I personally created that attempt to calculate the namespace and classname from the path to the file making assumptions that don't always prove true. I had never thought of just opening up and reading the file itself.

Reply:
Upon further review, if it were mine, I'd split that method out into its own class. Maybe in its own package. It's so elegantly done and so useful for so many things. It deserves a bigger life than that of a little protected function in the bottom of a file picker.

I've learned a ton diving into your code. This little PR has been fun for me.

Reply:
Hey, good points. For now I just made the method public on the Classfinder and you should now be able to use it. I see that it could be useful in its own class or package. Maybe I will do that some other time :-)

Reply:
That worked great. Thank you!